### PR TITLE
Cutting down heaps

### LINKED ISSUE #26: Possible effects if project fails
E.g. Potential benefits to the call center (in terms of reducing cost and being able to carry out more effective calls). 

500 words (including competitive advantages)

### PR DESCRIPTION
**Pull request by zachartmann**
Closes #26
* Cut down adv/disadv
* Cut down diagram descriptions
* Cut down ideas/reflection
* Removed report guide

### CODE REVIEW

**Comment by zachartmann** [comment]
I went buck wild so if I cut too much like an important line let me know

File: Report.md [hunk 1]
```diff
@@ -150,28 +151,27 @@ Sprint Burndown Chart | Graphical representation of development progress over ti
...
-This diagram represents the structure of the classes involved in the wider system. It displays their attributes and methods as well as the relationships that are between them, such as below:
-
+Class diagrams represent the structure of the classes involved in the system. It displays their attributes, methods and relationships between classes, such as below:
 ![Image of Class Diagram](./diagrams/ClassDiagram.png)
 
 ## **Collaboration Diagrams**
-These diagrams represents the dynamic behaviour of objects and the structure of the interactions between them. It also displays the sequence of these interactions through numbering; however, focuses more on the messages that flow between each object rather than time-based interactions (seen in sequences diagrams). The diagrams below show this in regards to outbound and inbound calls respectively:
+Collaboration diagrams represent the dynamic behaviour of objects and the structure of the interactions between them. It also displays the sequence of the interactions, focusing more on the messages that flow between objects than time-bised interactions. The collaboration diagrams for outbound and inbound calls, respectively, can be seen below:
```
Comment (seant1) [inline Report.md]:
```suggestion
Collaboration diagrams represent the dynamic behaviour of objects and the structure of the interactions between them. It also displays the sequence of the interactions, focusing more on the messages that flow between objects than time-based interactions. The collaboration diagrams for outbound and inbound calls, respectively, can be seen below:
```

**Review by seant1 — APPROVED** [review]
Typo

File: Report.md (hunk 1 above)
Comment (zachartmann) [reply Report.md]: Grammarly has failed me

**Pull request merged**